### PR TITLE
Cleaned up os/Time

### PR DESCRIPTION
**Pull request by marcusnaslund**
Minor style fixes. Peer review @thomasfanell ?

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Or maybe @sebastianbaginski ?

File: source/sdk/Time.ooc [hunk 1]
```diff
@@ -85,95 +84,93 @@ Time: class {
 
 	// The microseconds that have elapsed in the current minute.
 	microtime: static func -> LLong {
-		return microsec() as LLong + (sec() as LLong) * 1_000_000
+		microsec() as LLong + (sec() as LLong) * 1_000_000
```
Comment (sebastianbaginski) [inline source/sdk/Time.ooc]: I think it could be `microsec() + sec() * 1_000_000LL`

Comment (marcusnaslund) [inline source/sdk/Time.ooc]: It should also be `this ...`

File: source/sdk/Time.ooc [hunk 2]
```diff
@@ -85,95 +84,93 @@ Time: class {
...
-			// I believe every computer that runs ooc programs on Win32
-			// has a hardware high-performance counter, so it shouldn't be an issue
 			counter, frequency: LargeInteger
 			QueryPerformanceCounter(counter&)
 			QueryPerformanceFrequency(frequency&)
-			return ((counter quadPart * 1000) / frequency quadPart) - __time_millisec_base
+			result = ((counter quadPart * 1000) / frequency quadPart) - __time_millisec_base
 		} else {
-			return This runTimeMicro() / (1000 as UInt)
+			result = This runTimeMicro() / (1000 as UInt)
```
Comment (sebastianbaginski) [inline source/sdk/Time.ooc]: `result = This runTimeMicro() / 1000U`
Is it better ?

Comment (marcusnaslund) [inline source/sdk/Time.ooc]: It is, thanks, I will fix it in other places too.

**Comment by marcusnaslund** [comment]
Updated.

**Pull request merged**